ldap/servers/slapd/conntable.c

```diff
@@ -137,10 +129,8 @@ connection_table_new(int table_size)
     ct->fd = (struct POLL_STRUCT **)slapi_ch_calloc(1, ct->list_num * sizeof(struct POLL_STRUCT*));
     ct->table_mutex = PR_NewLock();
     /* Allocate the freelist */
-    ct->c_freelist = (Connection **)slapi_ch_calloc(1, ct->size * sizeof(Connection *));
-    /* NEVER use slot 0, this is a list pointer */
-    ct->conn_next_offset = 1;
-    ct->conn_free_offset = 1;
+    ct->c_freelist = (Connection **)slapi_ch_calloc(1, (ct->size+1) * sizeof(Connection *));
```

Comment:
Needs a comment that size here needs the +1 to be greater than the number of connections avaliable due to null terminate of the freelist.

Reply:
Fixed.